deepfence_server/handler/scan_reports.go

```diff
@@ -251,6 +251,36 @@ func (h *Handler) StartVulnerabilityScanHandler(w http.ResponseWriter, r *http.R
 	}
 }
 
+func (h *Handler) CompareScanHandler(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+	var req model.ScanCompareReq
+	err := httpext.DecodeJSON(r, httpext.NoQueryParams, MaxPostRequestSize, &req)
+	if err != nil {
+		log.Error().Msgf("%v", err)
+		h.respondError(&BadDecoding{err}, w)
+	}
+	if req.ScanType == "" {
+		h.respondError(incorrectScanTypeError, w)
+		return
+	}
+
+	if len(req.ScanIds) != 2 {
+		log.Error().Msgf("invalid scan ids: %s", req.ScanIds)
+		h.respondError(errors.New("could not compare, invalid scan ids"), w)
+		return
+	}
```

Comment:
If we can only have 2 ids, why are we using an array?
The API will be friendlier if we restrict it to 2 diff only, let's have `PrevScanId` & `NextScanId` instead of an array.